lib/ansible/modules/database/postgresql/postgresql_publication.py

```diff
@@ -61,6 +61,7 @@
     description:
     - Drop publication dependencies. Has effect with I(state=absent) only.
     type: bool
+    default: false
```

Comment:
Now you also have to add this to `argument_spec`.

Reply:
exactly! thanks